include/fs/fs.h

```diff
@@ -442,66 +447,66 @@ int fs_closedir(struct fs_dir_t *zdp);
  *
  * @param mp Pointer to the fs_mount_t structure
  *
- * @retval 0 Success
- * @retval -ERRNO errno code if error
+ * @retval 0 on success;
+ * @retval <0 negative errno code on error.
  */
 int fs_mount(struct fs_mount_t *mp);
 
 /**
  * @brief Unmount filesystem
  *
- * Perform steps needed for unmounting a file system like
+ * Perform steps needed to unmount a file system like
  * calling the file system specific unmount function and removing
  * the mount point from mounted file system list.
  *
- *
  * @param mp Pointer to the fs_mount_t structure
  *
- * @retval 0 Success
- * @retval -ERRNO errno code if error
+ * @retval 0 on success;
+ * @retval <0 negative errno code on error.
  */
 int fs_unmount(struct fs_mount_t *mp);
 
 /**
- * @brief Mount point read entry
+ * @brief Get path of mount point at index
  *
- * Read mount point entry
+ * Function iterates through list of mount points and returns directory name
+ * of mount point at given index.
+ * On success @p index is incremented and @p name is set to mount directory
+ * name.  If mount point with given @p index does not exist, @p name will be
+ * assigned @c NULL value
  *
- * @param number Pointer to mount point number
- * @param name Pointer to mount point name
+ * @param index Pointer to mount point index
+ * @param name Pointer to pointer to path name
  *
- * @retval 0 Success
- * @retval -ERRNO errno code if error
- * @return On success \p number is incremented and \p name is set to mount
- * point name. In case no mount point exists for the given \p number
- * -ENOENT is returned and \p name is set to NULL.
+ * @retval 0 on success;
+ * @retval -ENOENT if there is no mount point with given index.
  */
 int fs_readmount(int *number, const char **name);
 
 /**
  * @brief File or directory status
  *
- * Checks the status of a file or directory specified by the path
+ * Checks the status of a file or directory specified by the @p path
  *
  * @param path Path to the file or directory
  * @param entry Pointer to zfs_dirent structure to fill if file or directory
  * exists.
  *
- * @retval 0 Success
- * @retval -ERRNO errno code if error
+ * @retval 0 on success;
+ * @retval <0 negative errno code on error.
  */
 int fs_stat(const char *path, struct fs_dirent *entry);
 
 /**
- * @brief Retrieves statistics of the file system volume
+ * @brief statistics of the file system volume
```

Comment:
I think the original is better as it says what the function *does*.

Reply:
@de-nodic facepalms trying to figure out how he did missed that.